testing/src/main/java/io/stargate/it/http/RestApiv2SchemaTest.java

```diff
@@ -1114,9 +1114,9 @@ public void indexCreateInvalid() throws IOException {
                 "%s/v2/schemas/keyspaces/%s/tables/invalid_table/indexes",
                 restUrlBase, keyspaceName),
             objectMapper.writeValueAsString(indexAdd),
-            HttpStatus.SC_NOT_FOUND);
+            HttpStatus.SC_BAD_REQUEST);
     ApiError response = objectMapper.readValue(body, ApiError.class);
-    assertThat(response.getCode()).isEqualTo(HttpStatus.SC_NOT_FOUND);
+    assertThat(response.getCode()).isEqualTo(HttpStatus.SC_BAD_REQUEST);
```

Comment:
v1 has inconsistent rules here, missing table on an index creation is a 404, but for column operations it's a 400. Either we make the behavior consistent, or we'll need a way to tell `queryWithTable` which error to throw.

Reply:
Hmmmh. It seems like `404` would be better, but I guess 400 is ok too.

Maybe file a separate issue for resolving this discrepancy but use whatever logic is easiest here, for now, to get the optimized schema handling working even if at expense of error code inconsistence (whether between SGv1/SGv2 REST, or across REST operations).